packages/taucmdr/cf/software/tau_installation.py

```diff
@@ -502,7 +506,7 @@ def check_env_compat(cls):
                                      "Unload the darshan module and try again.")
         if os.environ.get('PE_ENV', '').lower() == 'cray':
             raise ConfigurationError("TAU Commander cannot be used with Cray compilers. ",
-                                     "Replace PrgEnv-cray with PrgEnv-intel, PrgEnv-gnu, or PrgEnv-pgi and try again.")
+                                     "Replace PrgEnv-cray with PrgEnv-intel, PrgEnv-gnu, PrgEnv-nvhpc, or PrgEnv-pgi and try again.")
```

Comment:
```suggestion
                                     "Replace PrgEnv-cray with PrgEnv-intel, PrgEnv-gnu, PrgEnv-nvidia, or PrgEnv-pgi and try again.")
```

The name of the Cray environment module that provides the NVHPC compilers is PrgEnv-nvidia. See https://docs.nersc.gov/systems/perlmutter/